script.js

```diff
@@ -1,53 +1,101 @@
-const filePattern = /jpg|jpeg|gif|png/;
-const mbSizeInBytes = 1024 * 1024;
-const totalDiskSpace = mbsToBytes(100);
-const defaultUsage = mbsToBytes(85);
-let currentUsage = parseInt(localStorage['currentUsage']) || defaultUsage;
-const mbText = ' MB';
-const pickerOpts = {
-    multiple: true,
+const TOTAL_DISK_SPACE = 100 * 1024 * 1024;
+const DEFAULT_USAGE = 85 * 1024 * 1024;
+const CACHE_CURRENT_USAGE = 'currentUsage';
+const PICKER_OPTS = {
+  multiple: true,
 };
+let CURRENT_USAGE;
 
-async function getFile() {
-    const fileHandles = await window.showOpenFilePicker(pickerOpts);
+const initApp = () => {
+  CURRENT_USAGE = parseInt(localStorage[CACHE_CURRENT_USAGE]) || DEFAULT_USAGE;
 
-    for (const fileHandle of fileHandles) {
-        const file = await fileHandle.getFile();
+  document.addEventListener('DOMContentLoaded', function () {
+    const updateStorageLimitElement = () => {
+      document.getElementById('storage-limit').innerHTML = getSizeString(TOTAL_DISK_SPACE);
+    }
+
+    updateStorageLimitElement();
+    updateUsageDomElements();
+  });
+}
+
+const getFile = async () => {
+  const isFileFormatValid = (fileType) => {
+    const FILE_PATTERN = /jpg|jpeg|gif|png/;
+
+    if (FILE_PATTERN.test(fileType)) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  const isThereEnoughStorageLeft = (fileSize) => {
+    return CURRENT_USAGE + fileSize <= TOTAL_DISK_SPACE;
+  }
+
+  const storeFile = (file) => {
+    CURRENT_USAGE += file.size;
+    localStorage[CACHE_CURRENT_USAGE] = CURRENT_USAGE;
+  }
+
+  const fileHandles = await window.showOpenFilePicker(PICKER_OPTS);
 
-        if (!filePattern.test(file.type)) {
-            alert("File format isn't supported");
-            return;
-        }
+  for (const fileHandle of fileHandles) {
+    const file = await fileHandle.getFile();
 
-        if (currentUsage + file.size <= totalDiskSpace) {
-            currentUsage += file.size;
-            localStorage['currentUsage'] = currentUsage;
-            updateUsageDomElements();
-        } else {
-            alert('There is not enough space on the disk');
-        }
+    if (!isFileFormatValid(file.type)) {
+      alert("File format isn't supported");
+      return;
     }
+
```

Comment:
Good job here, guard close is an important consept that help us write more readable code.